scripts/corr2_dsim_control.py

```diff
@@ -12,32 +12,57 @@
 
 from corr2.dsimhost_fpga import FpgaDsimHost
 
+
 parser = argparse.ArgumentParser(description='Control the dsim-engine (fake digitiser.)',
                                  formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
```

Comment:
Style: We don't usually put new lines between the arguments, so to match the rest of the code it would be better if you removed them.